driver-scala/build.gradle

```diff
@@ -26,6 +26,7 @@ dependencies {
     testImplementation project(':bson').sourceSets.test.output
     testImplementation project(':driver-sync').sourceSets.test.output
     testImplementation project(':driver-core').sourceSets.test.output
+    testImplementation project(':driver-reactive-streams').sourceSets.test.output
```

Comment:
I'm not sure this is worth it.  All it does is couple the sleep static variable in the reactive streams version of SyncMongoClient to the scala version.

Non of the other code is coupled to it. So I think this coupling should be removed.

Reply:
On reflection we already couple to driver-sync - so no harm